bundles/org.openhab.binding.mielecloud/src/main/resources/OH-INF/thing/channelTypes.xml

```diff
@@ -292,13 +340,21 @@
 		<label>@text/channel-type.mielecloud.fridge_super_cool.label</label>
 		<description>@text/channel-type.mielecloud.fridge_super_cool.description</description>
 		<category>Switch</category>
+		<tags>
+			<tag>Switch</tag>
+			<tag>Mode</tag>
```

Comment:
Similar to https://github.com/openhab/openhab-addons/pull/18585#discussion_r2055613730.